pyproject.toml

```diff
@@ -43,5 +43,6 @@ disable_error_code = [
 [[tool.mypy.overrides]]
 module = [
     "scipy.*",
+    "vivarium_testing_utils",
```

Comment:
Can you make a jira ticket to type-hint vtu? We may never do it but just in case Rajan thinks it's worthwhile